wi-fi/lwip/main.c

```diff
@@ -241,6 +241,14 @@ static void wifi_ap_thread(void *arg)
 			break;
 		}
 
+		result = cyhal_sdio_start_irq_thread(cybsp_get_wifi_sdio_obj());
```

Comment:
What has to be initiated by `cybsp_wifi_init_primary` in order to start SDIO IRQ thread? Is it interrupt handler?

Reply:
Without this change, [`whd_bus_sdio_irq_handler`](https://github.com/phoenix-rtos/phoenix-rtos-lwip/blob/b1f893ad418d5d73898b464259908e826ba5ae30/wi-fi/whd/bus_protocols/whd_bus_sdio_protocol.c#L1313C30-L1313C30) crashes, because `bus_priv` is freed by [`whd_bus_sdio_detach`](https://github.com/phoenix-rtos/phoenix-rtos-lwip/blob/b1f893ad418d5d73898b464259908e826ba5ae30/wi-fi/whd/bus_protocols/whd_bus_sdio_protocol.c#L208), which is called by [`cybsp_wifi_deinit`](https://github.com/phoenix-rtos/phoenix-rtos-lwip/blob/b1f893ad418d5d73898b464259908e826ba5ae30/wi-fi/lwip/cybsp_wifi.c#L602) or [`cybsp_wifi_init_primary_extended`](https://github.com/phoenix-rtos/phoenix-rtos-lwip/blob/b1f893ad418d5d73898b464259908e826ba5ae30/wi-fi/lwip/cybsp_wifi.c#L534C11-L534C43).

On second thought, I think I should reword the commit message and make sure that the thread is killed before `whd_bus_sdio_detach` is called.